import_tracker/__main__.py

```diff
@@ -568,39 +50,12 @@ def main():
         help="Indent for json printing",
         default=None,
     )
-    parser.add_argument(
-        "--log_level",
-        "-l",
-        help="Log level",
-        default=os.environ.get("LOG_LEVEL", "error"),
-    )
-    parser.add_argument(
-        "--recursive",
-        "-r",
-        action="store_true",
-        help="Recursively perform tracking on all nested modules",
-        default=False,
-    )
     parser.add_argument(
         "--submodules",
-        "-u",
-        nargs="*",
-        default=None,
-        help="List of sub-modules to recurse on (only used when --recursive set)",
-    )
-    parser.add_argument(
-        "--num_jobs",
-        "-j",
-        type=int,
-        help="Number of workers to spawn when recursing",
-        default=0,
-    )
-    parser.add_argument(
-        "--side_effect_modules",
```

Comment:
my nit would be to raise error if someone passes these, to make sure existing users address things with new update